src/lib/pages/start-workflow.svelte

```diff
@@ -64,6 +64,38 @@
 
   let searchAttributes: SearchAttributeInput[] = [];
 
+  $: errorWorkflowDetails = extractWorkflowFromError(error);
+
+  function extractWorkflowFromError(errorMessage: string): {
+    workflowId?: string;
+    runId?: string;
+  } {
+    if (!errorMessage) return {};
+
+    let workflowId: string | undefined;
+    let runId: string | undefined;
+
+    const workflowParts = errorMessage.split('WorkflowId: ');
+    if (workflowParts.length > 1) {
+      const workflowPart = workflowParts[1].split(', RunId:')[0];
+      if (workflowPart) {
+        workflowId = workflowPart.trim();
+      }
+    }
+
+    const runParts = errorMessage.split('RunId: ');
+    if (runParts.length > 1) {
+      const runPart = runParts[1].slice(0, -1).trim();
+      if (runPart) {
+        runId = runPart;
+      }
+    }
+    return {
+      workflowId,
+      runId,
+    };
+  }
```

Comment:
This might be a little cleaner way to do this with regex

  function extractWorkflowFromError(errorMessage: string): {
    workflowId?: string;
    runId?: string;
  } {
    if (!errorMessage) return {};

    const match = errorMessage.match(/WorkflowId: (.+?), RunId: (.+?)\.?$/);
    if (!match) return {};

    return {
      workflowId: match[1].trim(),
      runId: match[2].trim(),
    };
  }